web/src/components/studies/CreateStudyForm.tsx

```diff
@@ -0,0 +1,371 @@
+    name: "nhsEngland",
+    control,
+  });
+
+  const onSubmit: SubmitHandler<CreateStudyValues> = (data) => {
+    console.log(data);
+    //todo: do the things
+  };
+  const getFieldsetClass = (step: number) =>
+    `${styles.fieldset} ${currentStep === step ? styles.visible : styles.hidden}`;
+
+  return (
+    <Dialog setDialogOpen={setCreateStudyFormOpen} className={styles["study-dialog"]} cy="create-study-form">
+      <h2>Create Study</h2>
+      <div className={styles["step-progress"]}>
+        <div
+          className={`${styles["step-dot"]} ${currentStep === 1 ? styles["active"] : ""} ${isValid ? styles.valid : ""}`}
+        ></div>
+        <div className={`${styles["step-dot"]} ${currentStep === 2 ? styles["active"] : ""}`}></div>
+        <div className={`${styles["step-dot"]} ${currentStep === 3 ? styles["active"] : ""}`}></div>
+      </div>
+
+      <form className={styles.form} onSubmit={handleSubmit(onSubmit)}>
+        {/* first step */}
+        <fieldset className={getFieldsetClass(1)}>
+          <legend>Study Name</legend>
+          <Label htmlFor="shortStudyName">
+            Short Study Name*:
+            <Controller
+              name="shortStudyName"
+              control={control}
+              rules={{
+                required: "This field is required",
+                maxLength: {
+                  value: 55,
+                  message: "Maximum 55 characters",
+                },
+                pattern: {
+                  value: /^[a-zA-Z0-9 ]+$/,
+                  message: "Do not include any special characters",
+                },
+              }}
+              render={({ field }) => <Input {...field} type="text" id="shortStudyName" />}
+            />
+            <HelperText>Maximum 55 characters, do not include any special characters</HelperText>
+            {errors.shortStudyName && (
+              <Alert type="error">
+                <AlertMessage>{errors.shortStudyName.message}</AlertMessage>
+              </Alert>
+            )}
+          </Label>
+
+          <Label htmlFor="longStudyName">
+            Long Study Name:
+            <Textarea id="longStudyName" {...register("longStudyName", { maxLength: 255 })} />
+          </Label>
+        </fieldset>
+
+        <fieldset className={getFieldsetClass(1)}>
+          <legend>Ownership</legend>
+          <Label htmlFor="owner">
+            Study Owner (PI):
+            <Input
+              type="email"
+              id="owner"
+              {...register("owner")}
+              readOnly={true}
+              value={username}
+              inputClassName={styles.readonly}
+            />
+            <HelperText>
+              If you are not the study owner, contact the owner and ask them to fill out this form on their account.
+            </HelperText>
+          </Label>
+
+          <Label htmlFor="admin">
+            Study Administrator:
+            <Input type="email" id="admin" {...register("admin")} />
```

Comment:
this needs some validation that's it's a UCL email. need to think a little carefully about this one too as we'll want to know user principals (e.g. ccxyz@ucl.ac.uk) rather than email aliases (e.g. alice.smith@ucl.ac.uk), so we don't end up with duplicate users in the db

Reply:
shall I do a light validation that it's an @ucl.ac.uk address and when we get the entra stuff included we could do a check (they'll have the rest of the form to contend with so I'm sure they can be occupied whilst this happens) that runs the email given against entra and checks if they're 1. full staff and 2. if it's their principal address, if not then update the value, maybe with a message that says "you entered alice.smith@ucl which matched with our records for ccxyz@ucl"